package.json

```
@@ -1,44 +1,62 @@
 {
   "name": "graasp-repo",
-  "author": "Graasp Association",
   "version": "0.1.0",
   "description": "Repository template for Graasp",
+  "homepage": "https://github.com/graasp/graasp-repo#readme",
+  "bugs": {
+    "url": "https://github.com/graasp/graasp-repo/issues"
+  },
+  "repository": "git@github.com:graasp/graasp-repo.git",
+  "license": "AGPL",
+  "author": "Graasp Association",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "hooks:uninstall": "husky uninstall",
     "hooks:install": "husky install",
+    "hooks:uninstall": "husky uninstall",
+    "lint": "eslint .",
+    "prepack": "yarn build",
     "pre-commit": "pretty-quick --staged && yarn lint",
     "prepare": "yarn build",
-    "prepack": "yarn build",
     "prettier:check": "prettier --check \"{src,test}/**/*.ts\"",
     "prettier:write": "prettier --write \"{src,test}/**/*.ts\"",
     "test": "jest  --forceExit --coverage --verbose",
-    "test:watch": "yarn test --watchAll",
-    "lint": "eslint ."
+    "test:watch": "yarn test --watchAll"
   },
-  "repository": "git@github.com:graasp/graasp-repo.git",
-  "license": "AGPL",
-  "bugs": {
-    "url": "https://github.com/graasp/graasp-repo/issues"
+  "resolutions": {
+    "fastify": "3.29.1"
+  },
+  "dependencies": {
+    "@fastify/cookie": "^6.0.0",
+    "@graasp/etherpad-api": "github:graasp/etherpad-api",
+    "@types/luxon": "3.0.2",
+    "http-status-codes": "2.2.0",
+    "luxon": "3.0.4",
+    "uuid": "9.0.0"
   },
-  "homepage": "https://github.com/graasp/graasp-repo#readme",
   "devDependencies": {
     "@commitlint/cli": "^13.1.0",
     "@commitlint/config-conventional": "^13.1.0",
+    "@graasp/sdk": "github:graasp/graasp-sdk#etherpad",
```

- TODO: when https://github.com/graasp/graasp-sdk/pull/69 is merged
- 
```suggestion
    "@graasp/sdk": "github:graasp/graasp-sdk",
```